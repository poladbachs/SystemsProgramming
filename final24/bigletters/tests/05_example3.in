Appropriate
power to the people
