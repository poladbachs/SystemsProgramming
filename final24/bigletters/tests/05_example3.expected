 ** 
*  *
**** ***  ***      *** 
*  * *  * *  *     *  *
*  * ***  ***  r o ***  r i a t e
     *    *        *   



***                          ***      *** 
*  *                         *  *     *  *
***  o w e r   t o   t h e   ***  e o ***  l e
*                            *        *   
